nixos/modules/services/video/mirakurun.nix

```diff
@@ -0,0 +1,138 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.mirakurun;
+  mirakurun = pkgs.mirakurun;
+
+  serverConfig = recursiveUpdate cfg.extraConfig {
+    inherit (cfg) logLevel port;
+    path = cfg.domainSocket;
+  };
+
+in
+  {
+    options = {
+      services.mirakurun = {
+        enable = mkEnableOption mirakurun.meta.description;
+
+        user = mkOption {
+          type = types.str;
+          default = "mirakurun";
+          description = "User account under which Mirakurun runs.";
+        };
+
+        group = mkOption {
+          type = types.str;
+          default = "video";
+          description = "Group which mirakurun belongs to.";
+        };
+
+        nice = mkOption {
+          type = types.ints.between (-20) 19;
+          default = -10;
+          description = "Nice value for Mirakurun.";
+        };
+
+        ioSchedulingClass = mkOption {
+          type = types.enum [ "none" "realtime" "best-effort" "idle" ];
+          default = "realtime";
+          description = "IO scheduling class for Mirakurun.";
+        };
+
+        ioSchedulingPriority = mkOption {
+          type = types.ints.between 0 7;
+          default = 7;
+          description = "IO scheduling priority for Mirakurun.";
+        };
+
+        logLevel = mkOption {
+          type = types.ints.between (-1) 3;
+          default = 2;
+          description = "Mirakurun log level";
+        };
+
+        domainSocket = mkOption {
+          type = with types; nullOr path;
+          default = "/var/run/mirakurun/mirakurun.sock";
+          description = "Path to domain socket used to communicate with Mirakurun.";
+        };
+
+        port = mkOption {
+          type = with types; nullOr ints.u16;
+          default = 40772;
+          description = "Port to listen on.";
+        };
+
+        extraConfig = mkOption {
+          type = types.attrs;
+          default = {};
+          example = {
+            highWaterMark = 25165824;
+            overflowTimeLimit = 30000;
+          };
+          description = "Extra options which are added to server.yml.";
+        };
+
+        tunerConfig = mkOption {
+          type = with types; nullOr (listOf attrs);
+          default = [
+            {
+              name = "dummy-tuner";
+              isDisabled = true;
+            }
+          ];
+          example = [
```

Comment:
You need to wrap this with `literalExample` and use a literal string instead of a nix value to make the docs render correctly. Same with `extraConfig`.